aries-site/src/examples/components/tooltip/ToolTipSimpleExample.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box, Button, Tip, Text } from 'grommet';
+
+export const ToolTipSimpleExample = () => {
+  return (
+    <Box align="center" justify="center" fill>
+      <Tip
```

Comment:
The gap between the Button and Tip seems to be too large. In Figma, I see 6px gap which is 'xsmall'.

Reply:
Let me check on the gap and dig into that a little more to make sure it's correct

Reply:
Tracking with https://github.com/grommet/hpe-design-system/issues/1449